digits/model/images/generic/views.py

```diff
@@ -176,7 +176,7 @@ def generic_image_model_create():
                     val_interval    = form.val_interval.data,
                     pretrained_model= pretrained_model,
                     crop_size       = form.crop_size.data,
-                    use_mean        = bool(form.use_mean.data),
+                    use_mean        = form.use_mean.data,
```

Comment:
would you like to add new tests in `digits/model/images/generic/test_views.py` too?